packages/editor/src/components/table-of-contents/style.scss

```diff
@@ -35,24 +35,21 @@
 }
 
 .table-of-contents__counts {
-	display: flex;
-	flex-wrap: wrap;
 	margin: 0;
-	margin-top: -$grid-unit-10;
 }
 
 .table-of-contents__count {
-	flex-basis: 25%;
 	display: flex;
-	flex-direction: column;
+	align-items: center;
+	justify-content: space-between;
 	font-size: $default-font-size;
 	color: $gray-900;
-	padding-right: $grid-unit-10;
 	margin-bottom: 0;
-	margin-top: $grid-unit-10;
 
-	&:nth-child(4n) {
-		padding-right: 0;
+	& + & {
```

Comment:
I think it is pretty neat, while also a bit cleaner and maybe more robust than adding an extra first-child selector.  However, I do agree that `& + &` will be a bit more difficult to understand if someone has not seen it before.  Maybe a short comment would help clarify?

Reply:
I'd prefer to have `& + &`. Personally I find it more readable though I understand it might be cryptic for someone who has never seen it before. Added a little comment, let me know if that's all good or we should change it to a less cryptic solution.